### DISCUSSION TITLE
Provider Build Program

### DISCUSSION THREAD

**Discussion by rodri-r** [akash-network/community#435] [Ideas]
Greg recently Tweeted a holiday project that he would be working on

https://twitter.com/gregosuri/status/1736198214468894985

![akashgpu](https://github.com/akash-network/community/assets/46242399/a51c5960-f596-4a79-b152-ffeaa8386106)

If this build does take place and is in fact, capable of paying for itself, it would be nice to see some sort of Provider Build Program.

A grant/loan for individuals to build the exact same PC or something very similar. The grant could be paid back with the AKT the machine would earn as a provider.

Possible requirements for being able to request this:

- Being an Insider for a certain period of time.
- Have a provider running for a certain period of time.
- Sign some sort of agreement guaranteeing that the machine will not be sold and that it will in fact be used to run an Akash Provider.

**Comment by ArielLeza** [comment]
Great idea, I've been working the last days on a similar evaluation for a business plan (small office gaming/GPU farm) to generate profit as a business model

I am eager to see the benchmarks, I'll be happy to help 

About having the provider sign an agreement for the build program, perhaps we can build a logic with a contract that distributes the portion of the grant over time, while confirming availability, uptime or any other parameter we define that the provider must obey to in order to continue gaining the rewards

**Reply by rodri-r** [reply]
Glad to read that you like the idea and that you are also looking into having such provider.

About the agreement and smartcontract, while it does sound good:

1- Akash blockchain does not support smartcontracts right now, so this would need to be implemented on another chain and I think it overcomplicates things.
2- The grant/loan would need to be given out completely at first, as it is likely that people requesting it, do not have the funds to do the build on their own.
3- I think a traditional agreement, with a simple KYC could be enough

**Comment by rodri-r** [comment]
To add a bit to this idea

I believe that a program like this, would benefit the network in several ways:

1. Onboarding new and more providers
2. Higher quality providers
3. Network would become more decentralized (in my case I would run such a build/provider from a new local internet connection at my home or office, thus not on VPS's from other cloud providers or leasing servers that are in data centers that would take time to get to or that we might not have access to 24/7)
4. Network would most likely get new providers from locations that we currently do not have providers at.  Thus giving more options to end users to choose from. 
5. Prices offered to end users would most likely go a little lower, as I believe such a build/provider would cost less to maintain than to lease servers or VPS's, even with an Internet connection for the specific provider and electricity.

**Comment by gosuri** [comment]
This is the build I'm experimenting with for now.

**Processor: **Intel® Core™ i9-14900K Processor (8X 3.20GHz + 16X 2.20GHz/36MB L3 Cache)

**Motherboard: **MSI MAG Z790 Tomahawk MAX WiFi - WiFi 7, ARGB Header (3), USB 3.2 Ports (2 Type-C, 8 Type-A), M.2 Slot (4)

**Memory: **64 GB [32 GB X2] DDR5-6000MHz Memory Module - Certified Major Brand Gaming Memory

**Video Card: **NVIDIA GeForce RTX 4090 - 24GB GDDR6X (DLSS 3.0 – AI-Powered Performance)

**Power Supply: **1300 Watt - High Power - 80 PLUS Gold Certified PCIe GEN 5

**Processor Cooling: **be quiet! Pure Loop 2 360mm Liquid Cooler

**Primary Storage: **4TB Samsung 990 PRO M.2 PCIe Gen 4 NVMe SSD -- Gen 4 Read: 7450MB/s; Write: 6900MB/s, Gen 3 Read: 3572MB/s; Write: 3333 MB/s

**Secondary Storage: **4TB Samsung 990 PRO M.2 PCIe Gen 4 NVMe SSD -- Gen 4 Read: 7450MB/s; Write: 6900MB/s, Gen 3 Read: 3572MB/s; Write: 3333 MB/s

**Reply by alexx855** [reply]
I love this idea!

What if we remove the requirement *Have a provider running for a certain period of time. *? This would incentivize providers in underserved countries (I think this is one of the awesome advantages of AKT that we cannot utilize today because there aren't enough providers available in some countries). We could then award the grant after they've been online as a provider for a certain period in those areas or consider it as another track/incentive in the future.

**Comment by rodri-r** [comment]
I still believe this would be a good idea to onboard new providers, improve some current providers and as Alex mentions it should help to get new providers in regions that there are no providers right now. 

In the Akash Discord, in the Providers Build channel there has been some chatting about building providers with some used parts instead of buying new ones. I think this could be a good idea to decrease the initial build funding, as well as for getting a quicker return on investment for both providers and the network.

**Reply by oiclid** [reply]
@rodri-r The provider build guide listed in the [docs](https://akash.network/docs/other-resources/experimental/hardware-and-software-recommendations/#provider-hardware) is out of date. Are there any plans to update it soon?
